pkg/kubelet/prober/prober.go

```diff
@@ -152,7 +152,7 @@ func (pb *prober) runProbe(ctx context.Context, probeType probeType, p *v1.Probe
 	case p.Exec != nil:
 		klog.V(4).InfoS("Exec-Probe runProbe", "pod", klog.KObj(pod), "containerName", container.Name, "execCommand", p.Exec.Command)
 		command := kubecontainer.ExpandContainerCommandOnlyStatic(p.Exec.Command, container.Env)
-		return pb.exec.Probe(pb.newExecInContainer(ctx, container, containerID, command, timeout))
+		return pb.exec.Probe(pb.newExecInContainer(ctx, pod, container, containerID, command, timeout))
```

Comment:
you don't need to plumb down the entire pod object, just get the result of the pb.exec.Probe and log it here